activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1605,7 +1605,7 @@ def generate_index_name(table_name, column)
           name = "idx_on_#{Array(column) * '_'}"
 
           short_limit = max_index_name_size - hashed_identifier.bytesize
-          short_name = name.mb_chars.limit(short_limit).to_s
+          short_name = name.limit(short_limit).to_s
```

Comment:
```suggestion
          short_name = name.truncate_bytes(short_limit, omission: nil)
```

`String#limit` doesn't exist, and if it did, it wouldn't be in bytes.

Reply:
Thanks! fixed